compliance/azure/compliance_score/azure_regulatory_compliance_report.pt

```diff
@@ -97,6 +123,10 @@ script "js_calculate_score", type: "javascript" do
 EOS
 end
 
+##########
+# Policy
+##########
+
 policy "policy_azure_exec_reg_compliance" do
   validate $ds_format_report do
     summary_template "Regulatory Compliance Report"
```

Comment:
Please add and sort by Subscription Name

Reply:
Added and Sorted.